xonsh/completers/bash.py

```diff
@@ -116,14 +116,9 @@ def complete_from_bash(prefix, line, begidx, endidx, ctx):
     return out, len(prefix) - strip_len
 
 
-def _collect_completions_sources():
-    sources = []
+def _get_completions_source():
     completers = builtins.__xonsh_env__.get('BASH_COMPLETIONS', ())
-    paths = (pathlib.Path(x) for x in completers)
-    for path in paths:
+    for path in map(pathlib.Path, completers):
         if path.is_file():
-            sources.append('source "{}"'.format(path.as_posix()))
-        elif path.is_dir():
```

Comment:
@blahgeek I think this `elif` was mistakenly missed, and that's why bash completion now is not functional? :)